tests/tt_metal/tt_metal/unit_tests/common/device_fixture.hpp

```diff
@@ -77,3 +77,90 @@ class DeviceSingleCardFixture : public ::testing::Test {
     tt::ARCH arch_;
     size_t num_devices_;
 };
+
+class GalaxyFixture : public ::testing::Test {
+   protected:
+    void SkipTestSuiteIfSlowDispatchNotSet()
+    {
+        auto slow_dispatch = getenv("TT_METAL_SLOW_DISPATCH_MODE");
```

Comment:
We can remove this skip right?

Reply:
I'm not sure whether it can be removed. When I made this fixture, I copied the format of the other device fixtures:
https://github.com/tenstorrent/tt-metal/blob/main/tests/tt_metal/tt_metal/unit_tests/common/device_fixture.hpp
https://github.com/tenstorrent/tt-metal/blob/main/tests/tt_metal/tt_metal/unit_tests/common/n300_device_fixture.hpp

Reply:
Actually, I don't think it's needed. I've removed it so the fixtures should be able to run in fast dispatch or slow dispatch mode now.